iosApp/Modules/Screens/NowPlaying/UI/NowPlayingView.swift

```diff
@@ -24,6 +24,14 @@ public struct NowPlayingView: View {
 		}
 		.background(background)
 		.backButton()
+		//TODO: make an "ErrorProviding" protocol, replace all these.
```

Comment:
same here, can this be remved?

Reply:
I still need to add this back in but there were issues with the audio player errors I need to fix first.